### ISSUE TITLE
Continue download when navigating away from page

### ISSUE THREAD

**Issue by dachinat** [asapach/peerflix-server#147]
Hello, this is a really nice package and thank you for bringing it.

I have a question, perhaps a silly, did not dig into the internals yet.
Is it possible to continue downloading a file from torrent/magnet link after navigating away from actual page? Closing tab?

Thanks, I'd like to receive any help on this (maybe implementing if not supported.)

**Comment by asapach** [comment]
Sure, all the work is happening in the Node process, so it will keep downloading until you terminate it. Browser is just the UI.

**Comment by dachinat** [comment]
@asapach, thanks for replying. Right now when I click on Download button, a progress-bar will appear and an actual download will start (in Chrome downloads), however if I close the window where progress-bar appears, download terminates.

Is it something with my configuration or is this expected? I'd like to continue downloading file when that window is closed. Thanks.

**Comment by asapach** [comment]
I'm afraid I lost you:
> ...when I click on Download button, a progress-bar will appear and an actual download will start (in Chrome downloads)

You mean this Download button?
![image](https://user-images.githubusercontent.com/1755336/43034575-9594e1ea-8ce7-11e8-9e76-88123f9fb53f.png)
There should be nothing in Chrome downloads until you click the file link.

**Comment by dachinat** [comment]
@asapach Right, "when you click the file link", sorry my bad. So, download will start when you click the file link, however when I close a tab where I made a click, download stops, in Chrome download progress it freezes.

**Comment by asapach** [comment]
Generally I use `peerflix-server` for video, and as such I don't download the files, but instead use video streaming either in the browser for MP4, or using VLC: https://www.vlchelp.com/stream-online-videos-pc-vlc-media-player/

**Comment by asapach** [comment]
I've just tried downloading in Chrome anyway: when you close the *tab* it keeps downloading - it can take a while, because, you know, peer-to-peer.
When you close the *browser*, there's nothing I can do to keep the download running, sorry.
